README.md

```diff
@@ -93,13 +93,17 @@ When deploying the application, you can configure it to support different use ca
    $ echo "One way would be to poll s3 key for presence of .pem or .err (should a failure happen) files" 
 
    $ aws s3 cp s3://matterstackpai-dacinputs3tosqss3bucket<remainder of your bucket name>/arn:aws:acm-pca:<region>:<account>:certificate-authority/<PAI UUID>/<PID>/cert.pem .
+
+   $ echo "Change the DAC validity as needed"
+   $ aws lambda update-function-configuration --function-name <DAC_ISSUING_LAMBDA_NAME> --environment 'Variables {dacValidityInDays=<VALIDITY_IN_DAYS>}'
```

Comment:
Have you checked that it's safe to remove whatever variables are being added by default (as this command removes them all)?

Reply:
I don't think there are any variables being added be default. This is the only env var when i first create the lambda. But also yes I did run the cli command to update the env var and it still worked